includes/context/RequestContext.php

```diff
@@ -203,27 +203,64 @@ public function setUser( User $u ) {
 	public function getUser() {
 		// Wikia change - begin - @author: Michał Roszka
 		global $wgEnableHeliosExt;
+		$authPath = [];
+
 		if ( $this->user === null && $wgEnableHeliosExt ) {
 			$this->user = \Wikia\Helios\User::newFromToken( $this->getRequest() );
+			$authPath['helios'] = ($this->user !== null ? 'OK' : 'FAIL');
```

Comment:
I wonder if this should be `$authType = "helios"`. Then [below](https://github.com/Wikia/app/pull/8761/files#diff-f3b309ab0fd7d1151c05920cae0462ddR253) we could use `auth` as a facet in ES. As it is now, it's hard to graph in kibana.